### PR TITLE
Add json column representation

### PR DESCRIPTION
**Pull request by HLWeil**
#93

### CODE REVIEW

File: src/FsSpreadsheet/FsSpreadsheet.fsproj [hunk 1]
```diff
@@ -57,8 +58,6 @@
     <PackageReference Include="Thoth.Json.Core" Version="0.2.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Content Include="*.fsproj; **\*.fs; **\*.fsi" PackagePath="fable\" />
-  </ItemGroup>
+  <ItemGroup />
```
Comment (Freymaurer) [inline src/FsSpreadsheet/FsSpreadsheet.fsproj:61]: deleted fable files !!!

**Review by Freymaurer — CHANGES_REQUESTED** [review]

**Comment by Freymaurer** [comment]
did you verify against existing standards? https://spreadsheet.dsl.builders

**Comment by HLWeil** [comment]
Trying to verify against this example: https://spreadsheet.dsl.builders/#_sheets_and_rows

Encountered some issues. Mostly fixed but one Fable issue remains. Will link it here when I open the Issue there.

**Comment by HLWeil** [comment]
Should still be mergeable here IMO.

**Review by Freymaurer — APPROVED** [review]

**Pull request merged**